### DISCUSSION TITLE
__webpack_require__ is not defined

### DISCUSSION THREAD

**Discussion by lcas-k** [preactjs/preact-cli#1626] [Q&A]
Hello, I've got a problem, that if I create a Preact project I get the Error: 
`Uncaught ReferenceError: __webpack_require__ is not defined` 
and the whole page is blank even though I changed nothing, and it is just the pure installation.

Has anyone else had this issue and does someone know how to fix it?

Already, thanks in advance for any sort of help.

**Comment by JoviDeCroock** [comment]
I think this might be more appropriate for the preact-cli repo

**Reply by lcas-k** [reply]
Oh yeah, you're right, can I delete this discussion here somehow?

**Comment by rschristian** [comment]
@LucaswithC Where are you seeing this error? Dev, prod build? Which template are you using to initialize your project? Have you installed `preact-cli` globally, i.e., running `preact create ...`?

Need a bit more info, as I cannot reproduce.
